pkg/kubelet/kubelet.go

```diff
@@ -2388,13 +2388,17 @@ func (kl *Kubelet) isOutOfDisk() bool {
 	// Check disk space once globally and reject or accept all new pods.
 	withinBounds, err := kl.diskSpaceManager.IsRuntimeDiskSpaceAvailable()
 	// Assume enough space in case of errors.
-	if err == nil && !withinBounds {
+	if err != nil {
+		glog.Errorf("failed to check if disk space is available for the runtime: %v", err)
```

Comment:
I do have concerns about switching to always failing, but it also doesn't make sense that kubelet admits pods blindly without any information about disk usage. Let's punt the change for now until we have a better solution.